docs: send image file to the api endpoint (#4902)
langflow-ai/langflow: docs/docs/Workspace/workspace-api.md

The **Tweaks** tab displays the available parameters for your flow. Modifying the parameters changes the code parameters across all windows. For example, changing the **Chat Input** component's `input_value` will change that value across all API calls.

## Send image files to your flow with the API

> Send image files to the Langflow API.

Reviewer: This is pretty much a repeat of the title. Can you expand on this sentence with some information about why you would want to do this?

Author: Added "for AI analysis"

> The default file limit is 100 MB. This value can be configured by changing the `LANGFLOW_MAX_FILE_SIZE_UPLOAD` environment variable.

Reviewer: ```suggestion
The default file limit is 100 MB. To configure this value, change the `LANGFLOW_MAX_FILE_SIZE_UPLOAD` environment variable.
```

> For more, see [Environment variables](/environment-variables#supported-variables).

Reviewer: ```suggestion
For more information, see [Environment variables](/environment-variables#supported-variables).
```

1. To send an image to your flow with the API, POST the image file to the `v1/files/upload` endpoint of your flow.

```curl
curl -X POST "http://127.0.0.1:7860/api/v1/files/upload/a430cc57-06bb-4c11-be39-d3d4de68d2c4" \
  -H "Content-Type: multipart/form-data" \
  -F "file=@image-file.png"
```

> 2. The API returns the image file path in the format `"file_path":"<flow-id>/<timestamp>_<file-name>"}`.

Reviewer: This is a result, not an action that the user has to take, so it should not be a numbered step.

```json
{"flowId":"a430cc57-06bb-4c11-be39-d3d4de68d2c4","file_path":"a430cc57-06bb-4c11-be39-d3d4de68d2c4/2024-11-27_14-47-50_image-file.png"}
```

3. Post this image file to the **Chat Input** component of a **Basic prompting** flow.
Pass the file path value as an input in the **Tweaks** section of the curl call to Langflow.

```curl
curl -X POST \
    "http://127.0.0.1:7860/api/v1/run/a430cc57-06bb-4c11-be39-d3d4de68d2c4?stream=false" \
    -H 'Content-Type: application/json'\
    -d '{
    "output_type": "chat",
    "input_type": "chat",
    "tweaks": {
  "ChatInput-b67sL": {
    "files": "a430cc57-06bb-4c11-be39-d3d4de68d2c4/2024-11-27_14-47-50_image-file.png",
    "input_value": "what do you see?"
  }
}}'
```

> 4. Your chatbot describes the image file you sent.

Reviewer: This is a result, not an action that the user has to take, so it should not be a numbered step.
